### PR TITLE
nm, rm, xz: add Dutch translation

### PR DESCRIPTION
**Pull request by sebastiaanspeck**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Comment by Waples** [comment]
As a dutchy, I don't share your LGTM @kbdharun. There are typo's and some of the descriptions are not really typical Dutch, or to much Lazy dutch. @sebastiaanspeck can you update this?

File: pages.nl/common/nm.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+- Toon globale (externe) functies in een bestand (voorafgegaan door T):
+
+`nm -g {{pad/naar/bestand.o}}`
+
+- Toon alleen ongedefinieerde symbolen in een bestand:
+
+`nm -u {{pad/naar/bestand.o}}`
+
+- Toon alle symbolen, ook debugging symbolen:
```
Comment (leonvsc) [inline pages.nl/common/nm.md:14]: I am not sure if this is a nice sentence to read.

File: pages.nl/common/nm.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+`nm -a {{pad/naar/bestand.o}}`
+
+- Ontwar C++ symbolen (maak ze leesbaar):
```
Comment (leonvsc) [inline pages.nl/common/nm.md:18]: Same for this, what do you mean with `ontwar`?

File: pages.nl/common/rm.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+> Verwijder bestanden of mappen.
+> Zie ook: `rmdir`.
+> Meer informatie: <https://www.gnu.org/software/coreutils/rm>.
+
+- Verwijder specifieke bestanden:
+
+`rm {{pad/naar/bestand1 pad/naar/bestand2 ...}}`
+
+- Verwijder specifieke bestanden, maar negeer niet-bestaanden:
```
Comment (leonvsc) [inline pages.nl/common/rm.md:11]: I am not sure what the English sentence was but what do you mean with `maar negeer niet-bestaanden:`

File: pages.nl/common/xz.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+# xz
+
+> Comprimeren of decomprimeren van `.xz` en `.lzma` bestanden.
+> Meer informatie: <https://manned.org/xz>.
+
+- Comprimeer een bestand gebruik makend van xz file xz:
```
Comment (leonvsc) [inline pages.nl/common/xz.md:6]:
```suggestion
- Comprimeer een bestand gebruik makend van xz file:
```

I think you mean this?

File: pages.nl/common/xz.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+- Decomprimer een bestand en schrijf het naar `stdout` (impliceert `--keep`):
+
+`xz --decompress --stdout {{pad/naar/bestand.xz}}`
+
+- Comprimeer een bestand, maar verwijder het origineel niet:
+
+`xz --keep {{pad/naar/bestand}}`
+
+- Comprimeer een bestand gebruik makend van de snelste compressie:
```
Comment (leonvsc) [inline pages.nl/common/xz.md:30]:
```suggestion
- Comprimeer een bestand, gebruik makend van de snelste compressie:
```

File: pages.nl/common/xz.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+`xz -0 {{pad/naar/bestand}}`
+
+- Comprimeer een bestand gebruik makend van de bestee compressie:
```
Comment (leonvsc) [inline pages.nl/common/xz.md:34]:
```suggestion
- Comprimeer een bestand, gebruik makend van de beste compressie:
```

**Review by leonvsc — CHANGES_REQUESTED** [review]
See my suggestions and comments.

File: pages.nl/common/nm.md (hunk 1 above)
Comment (leonvsc) [reply pages.nl/common/nm.md:14]: I just read the documentation and I accept this sentence.

File: pages.nl/common/nm.md (hunk 2 above)
Comment (leonvsc) [reply pages.nl/common/nm.md:18]: I think this is better:

```suggestion
- Transformeer C++ symbolen (maak ze leesbaar):
```

**Review by leonvsc — APPROVED** [review]
1 suggestion left.

**Pull request merged**